core/modules/views/templates/views.theme.inc

```diff
@@ -453,10 +453,25 @@ function template_preprocess_views_view_table(&$variables) {
   $variables['rows'] = array();
   $variables['field_classes'] = array();
   $variables['header'] = array();
+  $variables['classes_array'] = array();
 
   $options  = $view->style_plugin->options;
   $handler  = $view->style_plugin;
 
+  if (!empty($handler->options['class'])) {
+    $classes = explode(' ', $handler->options['class']);
+    $classes = array_map('views_clean_css_identifier', $classes);
```

Comment:
This function doesn't exist in backdrop. Change this to `backdrop_clean_css_identifier()` and see if that works?

Reply:
Related: https://github.com/backdrop/backdrop-issues/issues/3716

Reply:
Thanks @jenlampton ...but hmmm 🤔 ...`backdrop_clean_css_identifier()` is in place of `drupal_clean_css_identifier()`, so wondering what the difference between that and `views_clean_css_identifier()` is. Will look into it.

Reply:
I noted it in the referenced issue.

Reply:
I have changed this to use `backdrop_clean_css_identifier()`, but was not smart enough to figure how to use `array_walk()` (which allows to pass a parameter to the callback function), so I have used a `foreach` instead. I then merged that with the `foreach` that was just below, as they both seemed to be iterating over the same array, and using the same key/values.

I hope I have done things right.